black.py

```diff
@@ -138,6 +138,23 @@ def from_configuration(
         return mode
 
 
+def abspath_pyproject_toml(path_search_start: str) -> Optional[str]:
```

Comment:
I feel like `find_pyproject_toml` would be a better name for this function. Also, why does it return a str instead of a Path?